assignment-client/src/avatars/AvatarMixerSlave.cpp

```diff
@@ -295,7 +295,7 @@ void AvatarMixerSlave::broadcastAvatarDataToAgent(const SharedNodePointer& node)
             : _avatar(avatar), _node(avatarNode), _lastEncodeTime(lastEncodeTime) {}
         glm::vec3 getPosition() const override { return _avatar->getClientGlobalPosition(); }
         float getRadius() const override {
-            glm::vec3 nodeBoxHalfScale = (_avatar->getClientGlobalPosition() - _avatar->getGlobalBoundingBoxCorner() * _avatar->getSensorToWorldScale());
+            glm::vec3 nodeBoxHalfScale = _avatar->getClientGlobalPosition() - _avatar->getGlobalBoundingBoxCorner();
```

Comment:
This is the bug which precipitated the need to purge bad uses of `sensorToWorldScale`.  In this case the code was doubly wrong: not only should the scale not be used at all, but it was being applied to only half of the difference, not the whole difference.

Again, the only reason this was never noticed was because most of the time the `sensorToWorldScale` is 1.0.